styles.css

```diff
@@ -0,0 +1,99 @@
+html {
+    font-size: 10px;
+  }
+  
+  body {
+
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto',
+      'Helvetica Neue', Arial, sans-serif;
+    font-size: 1.6rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+  }
```

Comment:
Kindly indent your code proper in line 6. Because this may lead you to have linter errors.